src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Tcp/ClientCredentialTypeTests.cs

```diff
@@ -174,4 +173,47 @@ public static void TcpClientCredentialType_Certificate_EchoString()
             ScenarioTestHelpers.CloseCommunicationObjects((ICommunicationObject)serviceProxy, factory);
         }
     }
+
+    [Fact]
+    [OuterLoop]
+    public static void TcpClientCredentialType_Certificate_CustomValidator_EchoString()
+    {
```

Comment:
Is there a negative version of this?   Sorry if I missed it, but I'd expect to see 2 tests -- 1 hit that endpoint with a valid certificate and a 2nd to hit it with an invalid one.